### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by SChandarana**
Had a bit of a git issue but its fixed now I think. Tables have been made and populated. The database is stored in SQLDB/Data and SQLDB/Logs (added into the gitignore). Is this okay or should it be stored somewhere external to this repository?

### CODE REVIEW

File: SQLDB/Queries/createTables.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+DROP TABLE Books;
+
+IF OBJECT_ID('LibraryBooks') IS NOT NULL
+DROP TABLE LibraryBooks;
+
+CREATE TABLE Users
+(
+	userId INT NOT NULL IDENTITY PRIMARY KEY,
+	username VARCHAR(32) NOT NULL,
+	userPw VARCHAR(32) NOT NULL,
```
Comment (FridaTveit) [inline SQLDB/Queries/createTables.sql]: Do you need to store this? You shouldn't store passwords as plain text. I believe the password sign in should be handled by the authentication you'll add later. If not you can always add more columns later if you need to 🙂

File: SQLDB/Queries/createTables.sql [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+);
+
+CREATE TABLE Loans
+(
+	userId INT NOT NULL,
+	bookId INT NOT NULL,
+	dueDate DATE NOT NULL,
```
Comment (FridaTveit) [inline SQLDB/Queries/createTables.sql]: You need to mark the primary key in this table and any foreign keys 🙂

File: SQLDB/Queries/insertData.sql [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	('978-1407104027'),
+	('978-1407104027'),
+	('978-1407104027'),
+	('978-1406360202'),
+	('978-0747532743'),
+	('978-0141346809');
+
+INSERT INTO dbo.Loans(userId, bookId, dueDate)
+VALUES
+	(1, 1, '2020-10-28')
```
Comment (FridaTveit) [inline SQLDB/Queries/insertData.sql:30]: To make sure these ids are correct you can set the ids when creating the users and books. To do that you need to set `IDENTITY_INSERT` to `ON` for the table you're editing, e.g.
```
SET IDENTITY_INSERT MyTable ON
INSERT INTO MyTable (Id, Name)
  VALUES (1, "Anna"), (2, "Bob")
SET IDENTITY_INSERT MyTable OFF
```
🙂

**Review by FridaTveit — COMMENTED** [review]
A few suggestions, otherwise looks good 🙂

**Comment by SChandarana** [comment]
Had to reorder the table creations as foreign key references stop dependent tables from being dropped

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**